Mathlib/NumberTheory/LSeries/Convolution.lean

```diff
@@ -83,52 +83,56 @@ lemma convolution_map_zero {R : Type*} [Semiring R] (f g : ℕ → R) : (f ⍟ g
 ### Multiplication of L-series
 -/
 
+/-- We give an expression of the `LSeries.term` of the convolution of two functions
+in terms of a sum over `Nat.divisorsAntidiagonal`. -/
+lemma term_convolution' (f g : ℕ → ℂ) (s : ℂ) (n : ℕ) :
```

Comment:
I feel the primed and non-primed versions should be the other way around.

Reply:
OK.

Reply:
You marked this as "resolved" but the naming didn't change – did you perhaps swap a different primed/unprimed lemma pair instead?

Reply:
I thought you meant `LSeries_convolution` and its primed variant; I guess I didn't read carefully.
Regarding `term_convolution`, the (currently) unprimed version is the one that is actually used, that's why I left it unprimed. If you think the other one is more useful outside, I can also change it there.
Should I revert the other changes? One version is in terms of `LSerieSummable`, the other in terms of abscissae of convergence, and I don't know whether there is a reason to consider one more canonical than the other...

Reply:
I feel the version with `sum` and `divisors_antidiagonal` is a more natural statement, hence my suggestion. But let's not spend too long debating over a single character! :-) I have no strong opinion about the other one.

Reply:
I've switched them now; I'm leaving the other ones.